namespaces/live.cloud-platform.service.justice.gov.uk/justice-gov-uk-production/resources/servicepod.tf

```diff
@@ -3,5 +3,5 @@ module "service_pod" {
 
   # Configuration
   namespace            = var.namespace
-  service_account_name = module.irsa.service_account.name # this uses the service account name from the irsa module
+  service_account_name = module.irsa.service_account_name # this uses the service account name from the irsa module
```

Comment:
It looks like this might is causing an error in concourse.

> This object does not have an attribute named "service_account_name".

Reply:
Thanks 👍